src/main/java/frc/robot/RobotContainer.java

```diff
@@ -77,9 +77,8 @@ public RobotContainer() {
     SmartDashboard.putData("LowerIntakeCommand", new MoveIntakeCommand(ArmState.LOWERED, intakeArms));
     SmartDashboard.putData("ForwardConveyorCommand", new MoveConveyorCommand(0.3, conveyor));
     SmartDashboard.putData("ReverseConveyorCommand", new MoveConveyorCommand(-0.3, conveyor));
-    SmartDashboard.putData("ForwardIntakeRollersCommand", new ForwardIntakeRollersCommand(intakeRollers));
-    SmartDashboard.putData("ReverseIntakeRollersCommand", new ReverseIntakeRollersCommand(intakeRollers));
-    SmartDashboard.putData("IntakeCommand", new IntakeCommand(intakeArms, conveyor, intakeRollers));
+    SmartDashboard.putData("ForwardIntakeRollersCommand", new MoveIntakeRollersCommand(0.3, intakeRollers));
+    SmartDashboard.putData("ReverseIntakeRollersCommand", new MoveIntakeRollersCommand(0.3, intakeRollers));
```

Comment:
Should these have the same value? They should probably be constants, as should the two values for the MoveConveyorCommand.

Reply:
No, they shouldn't. Good catch